Content.Server/Kitchen/EntitySystems/DeepFryerSystem.cs

```diff
@@ -0,0 +1,1004 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+using Content.Server.Administration.Logs;
+using Content.Server.Atmos.Components;
+using Content.Server.Atmos.EntitySystems;
+using Content.Server.Atmos.Miasma;
+using Content.Server.Body.Components;
+using Content.Server.Body.Systems;
+using Content.Server.Buckle.Components;
+using Content.Server.Cargo.Systems;
+using Content.Server.Chemistry.Components;
+using Content.Server.Chemistry.Components.SolutionManager;
+using Content.Server.Chemistry.EntitySystems;
+using Content.Server.Construction;
+using Content.Server.Construction.Components;
+using Content.Server.DoAfter;
+using Content.Server.Fluids.EntitySystems;
+using Content.Server.Ghost.Roles.Components;
+using Content.Server.Kitchen.Components;
+using Content.Server.NPC.Components;
+using Content.Server.Nutrition.Components;
+using Content.Server.Nutrition.EntitySystems;
+using Content.Server.Popups;
+using Content.Server.Power.Components;
+using Content.Server.Power.EntitySystems;
+using Content.Server.Temperature.Components;
+using Content.Server.Temperature.Systems;
+using Content.Server.UserInterface;
+using Content.Shared.Audio;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.Database;
+using Content.Shared.Destructible;
+using Content.Shared.Examine;
+using Content.Shared.FixedPoint;
+using Content.Shared.Hands.Components;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Interaction;
+using Content.Shared.Inventory;
+using Content.Shared.Item;
+using Content.Shared.Kitchen.Components;
+using Content.Shared.Kitchen.UI;
+using Content.Shared.MobState.Components;
+using Content.Shared.MobState.EntitySystems;
+using Content.Shared.Movement.Events;
+using Content.Shared.Nutrition.Components;
+using Content.Shared.Popups;
+using Content.Shared.Storage;
+using Content.Shared.Throwing;
+using Content.Shared.Tools.Components;
+
+namespace Content.Server.Kitchen.EntitySystems
+{
+    public sealed class DeepFryerSystem : EntitySystem
+    {
+        [Dependency] private readonly BloodstreamSystem _bloodstreamSystem = default!;
+        [Dependency] private readonly DamageableSystem _damageableSystem = default!;
+        [Dependency] private readonly DoAfterSystem _doAfterSystem = default!;
+        [Dependency] private readonly EntityManager _entityManager = default!;
```

Comment:
Isn't EntityManager always in scope in an entity system?

Reply:
You're right. I'll get this fixed.